kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js

```diff
@@ -76,7 +75,8 @@ define([
         );
     }
 
-    function renderParams(params) {
+    function renderParams(jobInfo) {
+        const params = jobInfo.job_params[0];
```

Comment:
jobInfo now stores all the job data from the backend (including app ID and app name), not just the job params